scripts/publish-storybook.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
```

Comment:
This is fantastic! In some ways should be its own npm package. We could create a new repo on https://github.com/mongodb-js and publish on there if we want. May add more overhead than we want right now.

Reply:
Probably more maintainable to rewrite in JS if it's not gonna be just a one-off and would also allow us to import the gh-pages package directly. One thing I wanted to do was make the storybook build happen it the background while you're prompted for username and such, but couldn't get ctrl+c to kill the child process. I imagine there must be something in npm that could make that easier.